src/main.ts

```diff
@@ -30,20 +30,26 @@ async function run() {
         await createTestAPI();
     }
     catch (err:any) {
+        console.log(err.message);
         core.setFailed(err.message);
     }
 }
 async function getTestAPI(validate:boolean) {
     var urlSuffix = "tests/"+testId+"?api-version=2023-04-01-preview";
```

Comment:
why are the files duplicated?

Reply:
this is not file, err.message only we can remove too, forgot to remove

Reply:
updated